### PR TITLE
Close #5 - Fix positioning when editor is scrolled

### LINKED ISSUE #5: Append control divs to Quill's parent element instead of document body?
Thanks for putting this module together--we were badly in need of a way to resize images in Quill and this really helped us out!

We have a single-page application with a Quill editor inside a scrollable container. Since the control boxes of the image resize module are appended to `document.body`, I found that scrolling the container would cause the resize boxes to drift away from their image:

screen shot 2017-04-06 at 11 32 41 am

To fix this, instead of appending the control divs to `document.body`, I append them to `quill.root.parentElement`. And instead of calculating their position using `this.img.getBoundingClientRect()`, I use `this.img.offsetLeft` and `this.img.offsetTop` to get positions relative to the container.

I'm not sure how generalizable this is, though. Would there be any drawbacks to always appending the control divs to `quill.root.parentElement`?

In any case, thanks a lot for your work on this module.

### PR DESCRIPTION
**Pull request by kensnyder**
- this is a proposed 3.0.0 version
- simplify toolbar and use Parchment Attributors
- use css float to align images
- add demo
- allow deleting an image
- update display when alignment changes
- allow referring to modules by name
- add .editorconfig
- fix lint errors
- update README
- avoid including all of lodash

### CODE REVIEW

**Comment by kensnyder** [comment]
@gbenson-wikia, will you review this PR and see what you think? This is my proposal for v3. See included demo or [the README's plunker](https://plnkr.co/edit/gq708AOrSBOWSlHcFslG?p=preview)

**Comment by gbenson-wikia** [comment]
Sorry for the slow response - I was out on vacation.

LGTM!  I'm happy to see this moving along so quickly.  :D

**Comment by kensnyder** [comment]
@gbenson-wikia Thanks for reviewing. This version is now published to npm as v3.0.0. If you find any shortcomings, feel free to open an PR or just let me know.

File: src/ImageResize.js [hunk 1]
```diff
@@ -114,6 +114,8 @@ class ImageResize {
             this.hideOverlay();
         }
 
+        this.quill.setSelection(null);
```
Comment (IanVS) [inline src/ImageResize.js:117]: This line is causing me some issues in my application, and I see that you're intentionally blurring the editor, but may I ask why?  I'm having trouble tracking this change back to the problem it solved.  Thanks!

**Pull request merged**